allennlp/nn/regularizers/regularizer_applicator.py

```diff
@@ -41,7 +41,7 @@ def __call__(self, module: torch.nn.Module) -> torch.Tensor:
         return accumulator
 
     @classmethod
-    def from_params(cls, params: List[Tuple[str, Params]]) -> 'RegularizerApplicator':
+    def from_params(cls, params: List[Tuple[str, Params]]) -> 'Optional[RegularizerApplicator]':
```

Comment:
Does `Optional['RegularizerApplicator']` work?  If not, that's fine, I just really dislike disabling unused imports, so if we can avoid it, that'd be better.